catalog/kubestash/raw/mariadb/mariadb-backup-function.yaml

```diff
@@ -12,4 +12,4 @@ spec:
   - --wait-timeout=${waitTimeout:=300}
   - --mariadb-args=${args:=}
   - --db-version=${dbVersion:=}
-  image: skaliarman/mariadb-restic-plugin:restic-backup_linux_amd64test12
+  image: ghcr.io/kubedb/mariadb-restic-plugin:v0.5.0
```

Comment:
Where is this `0.5.0` tag ?  I don't see any here https://github.com/kubedb/mariadb-restic-plugin/tags